### ISSUE TITLE
lint-staged throwing error

### ISSUE THREAD

**Issue by trodi** [angular-slider/angularjs-slider#555]
### Steps to reproduce
1. checkout latest code
2. `npm i --no-save`, `npm test`
3. `npm run precommit`

### Expected behaviour
In attempting to create a pull request, the `precommit` script is run. This error-ed out on my branch (details below). I checked out `master` and the same issue arises. It appears that something with the addition of the *prettier* causes this.

### Actual behaviour
```console
> lint-staged

`string` and `indent` should be strings
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! angularjs-slider@6.4.0 precommit: `lint-staged`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the angularjs-slider@6.4.0 precommit script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

#### Environment
`npm -v` => 5.2.0
`node -v` => 8.2.1

**Comment by ValentinH** [comment]
`string` and `indent` should be strings: do you know if it's related to your changes?

**Comment by trodi** [comment]
This happened even after I checked out a fresh clone, installed dependencies, and ran `precommit`.

Turns out this appears to be an issue with npm versions and the pack-lock file. I upgraded to npm 5.3, removed node_modules and the package-lock file and then `npm i` which fixed the issue. This will produced an updated package-lock file. Removing my node_modules again and `npm i` again leaving this new package-lock file, everything remains happy. We might want to update the package-lock file.

Here's the same issue for reference: https://github.com/okonet/lint-staged/issues/209

**Comment by ValentinH** [comment]
Thanks for taking the time to investigate this issue, that's really appreciated! :) Actually, I didn't even noticed that we had a package-lock file in this project... 

The issue is probably due to the fact that I didn't have an up-to-date version of npm when I set up prettier and thus it's dependencies did not get in the lock file .

**Comment by ValentinH** [comment]
Do you want to create a pull request with the updated list file?

**Comment by ValentinH** [comment]
PR merged ;)

**Comment by trodi** [comment]
`npm` automatically creates the lock file now.

Fixed by #556. Closing.
